<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1192</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_Actor_Section">
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="Lbl_Actor">
        <property name="text">
         <string>Actor Select</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_Actors">
        <property name="modelColumn">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Lbl_AddSource">
        <property name="text">
         <string>AddSource</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_Cube">
        <property name="text">
         <string>Cube</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_Sphere">
        <property name="text">
         <string>Sphere</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Lbl_Change">
        <property name="text">
         <string>Edit Source</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_Remove">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_Reset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="ActorSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="Lbl_Filter">
        <property name="text">
         <string>Filter Select</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Shrink">
        <property name="text">
         <string>Shrink Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="verticalWidget_Shrink" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="topMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="Lbl_Shrink_Filter">
           <property name="text">
            <string>Shrink Factor</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Clip">
        <property name="text">
         <string>Clip Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Elevation">
        <property name="text">
         <string>Elevation Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Transform">
        <property name="text">
         <string>Transform Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="verticalWidget_Transform" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <property name="topMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="Lbl_Transform">
           <property name="text">
            <string>Scale Factor</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_Transform">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_Scale_X">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>X</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_X_Transform">
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_Scale_Y">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Y</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_Y_Transform">
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_Scale_Z">
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Z</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBox_Z_Transform">
               <property name="minimum">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Btn_Filter">
        <property name="text">
         <string>Apply Filters</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="FilterSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_VTK_Widget">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QVTKOpenGLWidget" name="qvtkWidget_Tab1" native="true">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>400</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QVTKOpenGLWidget" name="qvtkWidget_Tab2" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Object_And_Project">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="HorizontalWidget_Position" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_Position">
           <property name="rightMargin">
            <number>1</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_Position">
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="Lbl_Position">
               <property name="text">
                <string>Object Position</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_Position_Spin_Boxes">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_Spinbox_X">
                 <item>
                  <widget class="QLabel" name="Lbl_X_Position">
                   <property name="text">
                    <string>X</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_X">
                   <property name="minimum">
                    <double>-99.989999999999995</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_Spinbox_Y">
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="Lbl_Y_Position">
                   <property name="text">
                    <string>Y</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_Y">
                   <property name="minimum">
                    <double>-99.989999999999995</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_Spinbox_X_2">
                 <item>
                  <widget class="QLabel" name="Lbl_Z_Position">
                   <property name="text">
                    <string>Z</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="doubleSpinBox_Z">
                   <property name="minimum">
                    <double>-99.989999999999995</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="Btn_Position">
               <property name="text">
                <string>Change Position</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_Model_Colour">
          <property name="text">
           <string>Change Model Colour</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_Change_Background">
          <property name="text">
           <string>Change Background Colour</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_Camera_Reset">
          <property name="text">
           <string>Reset Camera</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_Clear">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="ObjectSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1192</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFileOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileOpen"/>
   <addaction name="actionHelp"/>
  </widget>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="Icons/icons.qrc">
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
